src/shapes/group.class.js

```diff
@@ -349,7 +352,7 @@
       this._setObjectPosition(object);
 
       object.setCoords();
-      object.hasControls = object.__origHasControls;
+      object.hasControls = !!object.__origHasControls;
```

Comment:
If we can use `_updateObjectsCoords`, this fix is unnecessary.

If we cannot use `_updateObjectsCoords`, when we use `loadFromJSON` from json created by older version of fabric.js, _objects have no `__origHasControls`.
Therefore, I wish to not `object.hasControls = undefined`, but `object.hasControls = !!undefined = false`.